examples/packages/ssr/src/app.js

```diff
@@ -11,21 +12,33 @@ const Header = () => (
       height: 56,
       display: 'flex',
       alignItems: 'center',
+      '@media only screen and (max-width:1100px)': {
+        background: 'green',
+      },
     }}>
     server side rendering
   </header>
 );
 
+const Button = ({ count, setCount }) => (
```

Comment:
I added this to test the style consolidation of possibly 1000's of style elements was non-blocking